metricbeat/tests/metricbeat_test.py

```diff
@@ -1123,3 +1123,25 @@ def test_setting_fullnameOverride():
             "type": "DirectoryOrCreate",
         },
     } in volumes
+
+
+def test_adding_annotations():
+    config = """
+daemonset:
+    annotations:
+        foo: "bar"
+"""
+    r = helm_template(config)
+    assert "foo" in r["daemonset"][name]["metadata"]["annotations"]
+    assert r["daemonset"][name]["metadata"]["annotations"]["foo"] == "bar"
+    assert "annotations" not in r["deployment"][name + "-metrics"]["metadata"]
+    config = """
+deployment:
+    annotations:
+        grault: "waldo"
+"""
+    r = helm_template(config)
+    assert "grault" in r["deployment"][name + "-metrics"]["metadata"]["annotations"]
+    assert r["deployment"][name + "-metrics"]["metadata"]["annotations"]["grault"] == "waldo"
```

Comment:
Looks like there's one Python lint issue here:

```suggestion
    assert (
        r["deployment"][name + "-metrics"]["metadata"]["annotations"]["grault"]
        == "waldo"
    )
```

Reply:
@fatmcgav ...added the suggested change to `daemonset` and `deployment` test

Reply:
@kernkonzentrat It looks like the lint step is still not liking this format:

https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+lint-python/346/label=docker&&virtual&&ubuntu-18.04/console

Reply:
I formatted the tests with `black` now. I guess it will work eventually.